src/frontends/tensorflow/docs/tests.md

```diff
@@ -0,0 +1,48 @@
+# OpenVINO TensorFlow Frontend tests
+
+There are two types of tests for the TensorFlow Frontend (TF FE): layer tests and unit-tests.
+
+The layer tests are used to validate support of TensorFlow operation by the frontend.
+
+The unit-tests cover TensorFlow format reading functionality, conversion pipeline, and internal transformations
+for Transpose Sinking and conversion of sub-graphs with TF FE internal operations into OpenVINO opset.
+
+## How to build and run the unit-tests
+
+For building the TF FE unit-tests, use CMake target `ov_tensorflow_frontend_tests`. CMake automatically runs
+[generation scripts](../tests/test_models/gen_scripts) to create TensorFlow models used in the testing.
+
+Once the build is complete, launch `ov_tensorflow_frontend_tests` (`ov_tensorflow_frontend_tests.exe` for Windows)
+executable file to run all tests for the TensorFlow Frontend. The unit-tests use GoogleTest framework for execution.
+
+To get a tests coverage report for the TensorFlow frontend, read the page
+on [measuring coverage](../../../../docs/dev/test_coverage.md).
+
+## How to run the TensorFlow Frontend layer test
+
+The layer tests are Python-based and responsible to check that TensorFlow operation is supported by TF FE.
+The testing pipeline of the layer tests consists of three steps: create the single layer model with tested operation using TensorFlow,
```

Comment:
Let's make a list
The testing pipeline of the layer tests consists of three steps: 
1. Create the single layer model with tested operation using TensorFlow.
2. Convert this model into IR with TF FE.
3. Infer the original model using TensorFlow, infer the IR model using OpenVINO, and compare the inference results from both frameworks.

Reply:
Done